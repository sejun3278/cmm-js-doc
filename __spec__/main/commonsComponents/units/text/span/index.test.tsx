import "@testing-library/jest-dom";
import { render } from "@testing-library/react";

import _SpanText from "src/main/commonsComponents/units/text/span";
import { useRouter } from "next/router";

// 가짜 router 만들기
jest.mock("next/router", () => ({
  useRouter: jest.fn(),
}));
const push = jest.fn();
(useRouter as jest.Mock).mockImplementation(() => ({
  push,
}));

export default it("Text Span Tag Units Page Sanpshot", () => {
  const component = render(
    <_SpanText
      text="span 태그 테스트"
      className="_jest_span_text_"
      styles={{ color: "red" }}
    />
  );
  // @ts-ignore
  expect(component.container).toMatchSnapshot();
});
