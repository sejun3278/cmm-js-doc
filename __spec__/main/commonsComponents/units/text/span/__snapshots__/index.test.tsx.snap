// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Text Span Tag Units Page Sanpshot 1`] = `
<div>
  <span
    class="_spanText_ _jest_span_text_"
    style="color: red;"
  >
    span 태그 테스트
  </span>
</div>
`;
